### PR TITLE
Switch board to Morty

### PR DESCRIPTION
**Pull request by michal-mazurek**
Connect https://github.com/resin-os/meta-artik710/pull/12

### CODE REVIEW

File: layers/meta-resin-artik710/conf/samples/bblayers.conf.sample [hunk 1]
```diff
@@ -24,5 +24,5 @@ BBLAYERS_NON_REMOVABLE ?= " \
     ${TOPDIR}/../layers/poky/meta \
     ${TOPDIR}/../layers/poky/meta-yocto \
```
Comment (floion) [inline layers/meta-resin-artik710/conf/samples/bblayers.conf.sample]: this should be changed to meta-poky

Comment (floion) [inline layers/meta-resin-artik710/conf/samples/bblayers.conf.sample]: As well as also should be changed above in the BBLAYERS var declaration

**Comment by floion** [comment]
Why don't you also update meta-resin to latest master?

File: layers/meta-resin-artik710/recipes-kernel/linux/linux-yocto-artik710.bbappend [hunk 2]
```diff
@@ -3,5 +3,4 @@ inherit kernel-resin
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 SRC_URI_append = " \
     file://0001-s5p6818-artik710-raptor-common.dtsi-Add-gpio-leds-on.patch \
-    file://don-t-save-power.patch \
```
Comment (floion) [inline layers/meta-resin-artik710/recipes-kernel/linux/linux-yocto-artik710.bbappend]: Has the kernel changed? Why is this patch not apllying anymore?

Comment (michal-mazurek) [inline layers/meta-resin-artik710/recipes-kernel/linux/linux-yocto-artik710.bbappend]: The code changed significantly. I'd like to see if the issue still persists before porting the patch.

File: layers/meta-resin-artik710/conf/samples/bblayers.conf.sample (hunk 1 above)
Comment (floion) [inline layers/meta-resin-artik710/conf/samples/bblayers.conf.sample]: ping @michal-mazurek

File: layers/meta-resin-artik710/recipes-kernel/linux/linux-yocto-artik710.bbappend (hunk 2 above)
Comment (floion) [inline layers/meta-resin-artik710/recipes-kernel/linux/linux-yocto-artik710.bbappend]: Has there been a kernel update? I don't understand how the code changed as I don't think there was a kernel update in your PR

**Pull request merged**